### PR TITLE
chore: config updates

### PR DESCRIPTION
**Pull request by ximxim**
## Description

Updated parallel with latest assets and config
updating info upto blackbird partner drop

### CODE REVIEW

File: src/config/partners/anotherblock.ts [hunk 1]
```diff
@@ -16,14 +16,52 @@ const anotherblock: Partner = {
   twitter: '@anotherblock_io',
   drops: [
     {
+      startDate: Date.UTC(2023, 7, 12, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
+      endDate: Date.UTC(2023, 7, 14, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
```
Comment (janacm) [inline src/config/partners/anotherblock.ts:20]: Hey, end time is going to be 11:59 PM, not AM, so I believe `CAMPAIGN_HOUR` needs to be updated

File: public/partners/coke/drop/drop.png
Comment (janacm) [inline public/partners/coke/drop/drop.png:1]: We don't need this masterpiece collection image, we won't use it anywhere

File: src/config/partners/parallel.ts [hunk 2]
```diff
@@ -15,14 +15,38 @@ const parallel: Partner = {
...
+Each purchase also includes a Heroes of Parallel Cardback which can also be used in game.
+
+The drop is live now and will be available throughout Onchain Summer or until sold out. Each deck will be available for .05 ETH on Base."`,
+      address: '0x206571b68c66E1d112b74d65695043ad2b5F95D5',
+      crossMintClientId: '',
+      mintType: MintType.External,
+      externalLink: 'https://parallel.life/starter-decks',
+      type: 'erc-721',
+      startDate: Date.UTC(2023, 7, 11, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
+      endDate: Date.UTC(2023, 7, 31, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
```
Comment (janacm) [inline src/config/partners/parallel.ts:34]: Update end time to August 31st 11:59PM

File: src/config/partners/parallel.ts [hunk 3]
```diff
@@ -15,14 +15,38 @@ const parallel: Partner = {
...
+
+Each purchase also includes a Heroes of Parallel Cardback which can also be used in game.
+
+The drop is live now and will be available throughout Onchain Summer or until sold out. Each deck will be available for .05 ETH on Base."`,
+      address: '0x206571b68c66E1d112b74d65695043ad2b5F95D5',
+      crossMintClientId: '',
+      mintType: MintType.External,
+      externalLink: 'https://parallel.life/starter-decks',
+      type: 'erc-721',
+      startDate: Date.UTC(2023, 7, 11, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
```
Comment (janacm) [inline src/config/partners/parallel.ts:33]: Update start date to august

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**